packages/backend-common/api-report.md

```diff
@@ -359,7 +359,6 @@ export class GerritUrlReader implements UrlReader {
     deps: {
       treeResponseFactory: ReadTreeResponseFactory;
     },
-    workDir: string,
```

Comment:
Maybe call out this change in the changeset as well.

Reply:
Yes!